rules/windows/command_and_control_remote_file_copy_desktopimgdownldr.toml

```diff
@@ -32,7 +32,7 @@ event.category:process and event.type:(start or process_started) and
 framework = "MITRE ATT&CK"
 [[rule.threat.technique]]
 id = "T1105"
-name = "Remote File Copy"
```

Comment:
glad to see this name go.
weird that it was tagged as lateral movement and C2 before. totally different things